draft-ietf-rtcweb-jsep.xml

```diff
@@ -1901,15 +1970,20 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>A direction attribute which is the same as that of the
               associated transceiver.</t>
 
-              <t>For each supported codec, "a=rtpmap" and "a=fmtp"
-              lines, as specified in
+              <t>For each supported codec that has not been excluded by
+              the codec preferences of the associated transceiver,
+              "a=rtpmap" and "a=fmtp" lines, as specified in
               <xref target="RFC4566"></xref>, Section 6. The audio and
               video codecs that MUST be supported are specified in
               <xref target="I-D.ietf-rtcweb-audio"></xref>(see Section
               3) and
               <xref target="I-D.ietf-rtcweb-video"></xref>(see Section
               5).</t>
 
+              <t>If codec preferences have been set for the associated
```

Comment:
I think this needs to go into the discussion of the m= line generation itself (e.g. the section after "Each m= section should be generated as specified in [RFC4566],
   Section 5.14.  For the m= line itself, the following rules MUST be
   followed:")